src/utils/utils.cpp

```diff
@@ -606,6 +847,18 @@ QueryData ExecuteQuery(mg_session *session, const std::string &query) {
     }
   }
 
+  const mg_map *summary = mg_result_summary(result);
+  if (summary && mg_map_size(summary) > 0) {
+    const mg_value *mg_stats = mg_map_at(summary, "stats");
+    const mg_value *mg_notifications = mg_map_at(summary, "notifications");
+    if (mg_stats) {
```

Comment:
Consider the if statement with initializer:
```suggestion
    if (const mg_value *mg_stats = mg_map_at(summary, "stats"); mg_stats) {
```